src/frontend/apps/e2e/__tests__/app-desk/footer.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { expect, test } from '@playwright/test';
+
+import { keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Footer', () => {
+  test('checks all the elements are visible', async ({ page }) => {
+    const footer = page.locator('footer').first();
+
+    await expect(footer.getByAltText('Marianne Logo')).toBeVisible();
+
+    await expect(
+      footer.getByAltText('Freedom Equality Fraternity Logo'),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'legifrance.gouv.fr' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'info.gouv.fr' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'service-public.fr' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'data.gouv.fr' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'Legal Notice' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'Personal data and cookies' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByRole('link', { name: 'Accessibility' }),
+    ).toBeVisible();
+
+    await expect(
+      footer.getByText(
+        'Unless otherwise stated, all content on this site is under',
+      ),
+    ).toBeVisible();
+  });
+
+  const legalPages = [
+    { name: 'Legal Notice', url: '/legal-notice' },
+    { name: 'Personal data and cookies', url: '/personal-data-cookies' },
+    { name: 'Accessibility', url: '/accessibility' },
```

Comment:
As the app has not been audited for its accessibility, we need to use the name "accessibility: 
non-compliant" for this link. Like for the suite territoriale landing page's footer (https://suiteterritoriale.anct.gouv.fr/).

![Screenshot 2024-06-26 at 12 59 20](https://github.com/numerique-gouv/people/assets/33085077/ae1dd545-6dad-4ac2-a6c7-98f8893ab60c)